src/components/dialogmote/innkalling/DialogmoteInnkallingSkjema.tsx

```diff
@@ -178,6 +179,7 @@ const DialogmoteInnkallingSkjema = () => {
 
   return (
     <Box background="surface-default" padding="6" className="mb-2">
+      <ChooseMalform />
```

Comment:
Er det forresten sikkert at man ønsker samme målform i alle brevene?

Reply:
Godt poeng!

Reply:
Vi vet ikke, men hvis man bor på vestlandet så kan det jo tenkes at man vil at nynorsk skal være default hvis man setter det én gang 🤔

Reply:
Ja, bør kanskje lufte det for Stine eller noe. Ser fort for meg et behov for å kunne velge ulike målformer til de ulike partene også :confused:

Reply:
Ja, den har vi diskutert litt med henne. "Kravet" gjelder til arbeidstaker, men det kan hende man på sikt vil kunne velge målform for hver av brevene ja. 

Kanskje vi burde hatt noen målinger på om brevet er nynorsk/bokmål, @JMLindseth? I Amplitude feks.